external/asf4-drivers/qtouch/qtouch.c

```diff
@@ -604,17 +608,25 @@ void qtouch_process_scroller_positions(void)
 
         // Calculate scroller position using a moving average
         // The moving average wieghts previous N readings twice current reading
+        uint16_t estimated_scroller_position;
```

Comment:
Here to L629: What is the need for these edits? Looks like no functional change except for setting the position to zero if below threshold. However, I dont think it makes sense to set it to zero - makes more sense that `qtouch_get_scroller_position` reports the last scroller position that was hit. I dont remember off hand, but it is possible that other code relies on this value being the last position hit.

Reply:
Dropped to a separate branch.